### DISCUSSION TITLE
[DataTable]: header's padding are not aligned with the cell's padding when columnsGap and renderRow props are applied

### DISCUSSION THREAD

**Discussion by mari-shakhanava** [epam/UUI#2595] [Q&A] [answered]
## Steps to Reproduce

1. There is DataTable component with default props and following props applied
```
columnsGap="12"
size="30"
renderRow = {}
```
2.  textAlign: "left" for the first column

Example:
https://codesandbox.io/p/sandbox/uui-forked-l3f33k?file=%2FExample.tsx%3A106%2C7-108%2C11

## Actual result

Header's padding is not aligned with the cell's padding when columnsGap prop applied.
Cell has     --uui-dt-cell-padding-start: 24px;
Header cell has     --uui-dt-cell-padding-start: 12px;
![2024-10-25_13h00_51](https://github.com/user-attachments/assets/4dc276f7-57ac-4c02-96d8-58d4d7db3f2f)
![2024-10-25_12h58_15](https://github.com/user-attachments/assets/20c1b9e7-793e-4d7a-b1ae-06c6eb1897fb)

## Expected result

The header's and cell's text are aligned in the first column, as well as they are aligned in the other columns, or as well as they are aligned without the renderRow option passed to the DataTable props.
![2024-10-25_13h12_16](https://github.com/user-attachments/assets/73e74316-f2d9-4f5d-b5f4-ea7eca690489)

## Environment

* UUI version: latest
* Browser name: latest
* Operating System: windows

**Comment by AlekseyManetov** [comment, marked as answer]
If you provide `renderRow` callback, you also should specify `columnGap` prop for `DataTableRow`, otherwise default `columnGap` value will be used, which is equal to '24'.
<img width="683" alt="image" src="https://github.com/user-attachments/assets/3d376da4-f233-4c1a-9e8d-9288229cf8f8">
